app/controllers/api/admin/news/items_controller.rb

```diff
@@ -0,0 +1,94 @@
+module Api
+  module Admin
+    module News
+      class ItemsController < ApiController
+        before_action :authenticate_user!
+
+        def index
+          render json: serialize(news_items.to_a, pagination_meta)
```

Comment:
I'd ask for the news items rather than tell, here.

```suggestion
          render json: serialized_news_items
```

Reply:
:man_facepalming:

Reply:
Looking at this in more detail - serialize is used with multiple sources - both a single instance (eg #create) and arrays, (#index) so its not possible to have a general serialization method without giving it context (ie passing in the data to be serialized)

I'll leave this as is otherwise I'm getting into adding separate methods for all scenarios rather then generalising the common bit of code (choosing the serializer to use)